src/Microsoft.VisualStudio.ProjectSystem.Managed/Microsoft.VisualStudio.ProjectSystem.Managed.csproj

```diff
@@ -29,6 +29,13 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Tpl.Dataflow" Version="4.5.24" />
```

Comment:
I think the package references were a transitory problem I had. There was an issue with an older version of the project system getting pulled in and this forced it to the right version. I removed the references and everything seems OK so the problem must have passed.

---
In reply to: [117537469](https://github.com/dotnet/project-system/pull/2241#discussion_r117537469) [](ancestors = 117537469)